libraries/classes/Controllers/Database/SqlController.php

```diff
@@ -56,14 +56,15 @@ public function __invoke(ServerRequest $request): void
          */
         $GLOBALS['goto'] = Url::getFromRoute('/database/sql');
         $GLOBALS['back'] = $GLOBALS['goto'];
+        $delimiter = $request->getParsedBodyParam('delimiter');
 
         $this->response->addHTML($this->sqlQueryForm->getHtml(
             $GLOBALS['db'],
             '',
             true,
             false,
-            isset($_POST['delimiter'])
-                ? htmlspecialchars($request->getParsedBodyParam('delimiter'))
+            isset($delimiter)
```

Comment:
```suggestion
            $delimiter !== null
```
it is always isset ;)